Random failure in TestHarness/tests.write_results

## Rationale
This test seems to have failed randomly.  It failed on devel but then passed on new commits to devel. 

<!--What is the reason for this enhancement or what error are you reporting?-->

## Description

<!--Provide details of the enhancement or instructions for reproducing the error.-->
See: https://civet.inl.gov/job/171928/

```
TestHarness/tests.write_results: Working Directory: /opt/civet/build_0/moose/python/TestHarness/tests
TestHarness/tests.write_results: Running command: python -m unittest -v test_WriteResults
TestHarness/tests.write_results: testWriteAll (test_WriteResults.TestHarnessTester)
TestHarness/tests.write_results: Test write all output files --sep-files ... FAIL
TestHarness/tests.write_results: testWriteFail (test_WriteResults.TestHarnessTester)
TestHarness/tests.write_results: Test ability to write separate Fail test --sep-files-fail ... ok
TestHarness/tests.write_results: testWriteOK (test_WriteResults.TestHarnessTester)
TestHarness/tests.write_results: Test ability to write separate OK test --sep-files-ok ... ok
TestHarness/tests.write_results: 
TestHarness/tests.write_results: ======================================================================
TestHarness/tests.write_results: FAIL: testWriteAll (test_WriteResults.TestHarnessTester)
TestHarness/tests.write_results: Test write all output files --sep-files
TestHarness/tests.write_results: ----------------------------------------------------------------------
TestHarness/tests.write_results: Traceback (most recent call last):
TestHarness/tests.write_results:   File "test_WriteResults.py", line 62, in testWriteAll
TestHarness/tests.write_results:     self.fail('Failed to create all output files --sep-files')
TestHarness/tests.write_results: AssertionError: Failed to create all output files --sep-files
TestHarness/tests.write_results: 
TestHarness/tests.write_results: ----------------------------------------------------------------------
TestHarness/tests.write_results: Ran 3 tests in 9.678s
TestHarness/tests.write_results: 
TestHarness/tests.write_results: FAILED (failures=1)
TestHarness/tests.write_results: 
TestHarness/tests.write_results: ################################################################################
TestHarness/tests.write_results: Tester failed, reason: CRASH
TestHarness/tests.write_results: 
    FAIL TestHarness/tests.write_results FAILED (CRASH)
```

## Impact

<!--How will the changes impact the code, developers, and users? Discuss changes to the
internal interfaces and public API.-->
More robust testing of the TestHarness

Looks like it might be conflicting with `TestHarness/tests.diff` when run in parallel
since both tests use the same test spec `diffs`.
```
TestHarness/tests.diff: Working Directory: /opt/civet/build_0/moose/python/TestHarness/tests
TestHarness/tests.diff: Running command: python -m unittest -v test_Diff
TestHarness/tests.diff: testDiffs (test_Diff.TestHarnessTester) ... FAIL
TestHarness/tests.diff: 
TestHarness/tests.diff: ======================================================================
TestHarness/tests.diff: FAIL: testDiffs (test_Diff.TestHarnessTester)
TestHarness/tests.diff: ----------------------------------------------------------------------
TestHarness/tests.diff: Traceback (most recent call last):
TestHarness/tests.diff:   File "test_Diff.py", line 23, in testDiffs
TestHarness/tests.diff:     self.assertRegexpMatches(e.output, r'test_harness\.csvdiff.*?FAILED \(CSVDIFF\)')
TestHarness/tests.diff: AssertionError: Regexp didn't match: 'test_harness\\.csvdiff.*?FAILED \\(CSVDIFF\\)' not found in '\x1b[33mtests/test_harness.exodiff: \x1b[0mWorking Directory: /opt/civet/build_0/moose/test/tests/test_harness\n\x1b[33mtests/test_harness.exodiff: \x1b[0mRunning command: /opt/civet/build_0/moose/test/moose_test-opt -i exodiff.i --error --error-unused --error-override --no-gdb-backtrace\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0mFramework Information:\n\x1b[33mtests/test_harness.exodiff: \x1b[0mMOOSE Version:           git commit b22a48e on 2018-03-28\n\x1b[33mtests/test_harness.exodiff: \x1b[0mLibMesh Version:         727d0685dc96e841c5805608653410ea0cd926a6\n\x1b[33mtests/test_harness.exodiff: \x1b[0mPETSc Version:           3.7.6\n\x1b[33mtests/test_harness.exodiff: \x1b[0mCurrent Time:            Wed Mar 28 18:14:08 2018\n\x1b[33mtests/test_harness.exodiff: \x1b[0mExecutable Timestamp:    Wed Mar 28 17:52:22 2018\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0mParallelism:\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Num Processors:          1\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Num Threads:             1\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0mMesh: \n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Parallel Type:           replicated\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Mesh Dimension:          2\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Spatial Dimension:       2\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Nodes:                   \n\x1b[33mtests/test_harness.exodiff: \x1b[0m    Total:                 9\n\x1b[33mtests/test_harness.exodiff: \x1b[0m    Local:                 9\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Elems:                   \n\x1b[33mtests/test_harness.exodiff: \x1b[0m    Total:                 4\n\x1b[33mtests/test_harness.exodiff: \x1b[0m    Local:                 4\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Num Subdomains:          1\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Num Partitions:          1\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0mNonlinear System:\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Num DOFs:                9\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Num Local DOFs:          9\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Variables:               "u" \n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Finite Element Types:    "LAGRANGE" \n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Approximation Orders:    "FIRST" \n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0mExecution Information:\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Executioner:             Transient\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  TimeStepper:             ConstantDT\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Solver Mode:             Preconditioned JFNK\n\x1b[33mtests/test_harness.exodiff: \x1b[0m  Preconditioner:          hypre boomeramg \n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0mTime Step  0, time = 0\n\x1b[33mtests/test_harness.exodiff: \x1b[0m                dt = 0\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0mTime Step  1, time = 0.1\n\x1b[33mtests/test_harness.exodiff: \x1b[0m                dt = 0.1\n\x1b[33mtests/test_harness.exodiff: \x1b[0m 0 Nonlinear |R| = \x1b[32m1.732051e+01\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      0 Linear |R| = \x1b[32m1.732051e+01\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      1 Linear |R| = \x1b[32m6.094239e-01\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      2 Linear |R| = \x1b[32m3.918388e-03\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      3 Linear |R| = \x1b[32m3.330983e-15\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m 1 Nonlinear |R| = \x1b[32m1.414052e-06\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      0 Linear |R| = \x1b[32m1.414052e-06\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      1 Linear |R| = \x1b[32m4.975358e-08\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      2 Linear |R| = \x1b[32m3.198986e-10\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      3 Linear |R| = \x1b[32m4.460400e-18\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m 2 Nonlinear |R| = \x1b[32m1.773058e-14\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\x1b[32m Solve Converged!\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0mTime Step  2, time = 0.2\n\x1b[33mtests/test_harness.exodiff: \x1b[0m                dt = 0.1\n\x1b[33mtests/test_harness.exodiff: \x1b[0m 0 Nonlinear |R| = \x1b[32m1.640283e+00\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      0 Linear |R| = \x1b[32m1.640283e+00\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      1 Linear |R| = \x1b[32m2.793149e-02\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      2 Linear |R| = \x1b[32m4.100192e-04\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      3 Linear |R| = \x1b[32m1.538532e-14\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m 1 Nonlinear |R| = \x1b[32m7.439466e-09\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\x1b[32m Solve Converged!\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0mTime Step  3, time = 0.3\n\x1b[33mtests/test_harness.exodiff: \x1b[0m                dt = 0.1\n\x1b[33mtests/test_harness.exodiff: \x1b[0m 0 Nonlinear |R| = \x1b[32m1.464539e+00\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      0 Linear |R| = \x1b[32m1.464539e+00\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      1 Linear |R| = \x1b[32m2.493884e-02\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      2 Linear |R| = \x1b[32m3.660890e-04\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      3 Linear |R| = \x1b[32m3.384414e-14\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m 1 Nonlinear |R| = \x1b[32m4.049647e-09\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\x1b[32m Solve Converged!\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0mTime Step  4, time = 0.4\n\x1b[33mtests/test_harness.exodiff: \x1b[0m                dt = 0.1\n\x1b[33mtests/test_harness.exodiff: \x1b[0m 0 Nonlinear |R| = \x1b[32m1.307624e+00\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      0 Linear |R| = \x1b[32m1.307624e+00\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      1 Linear |R| = \x1b[32m2.226682e-02\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      2 Linear |R| = \x1b[32m3.268648e-04\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m      3 Linear |R| = \x1b[32m7.384183e-14\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m 1 Nonlinear |R| = \x1b[32m3.822944e-09\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\x1b[32m Solve Converged!\x1b[39m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0m -------------------------------------------------------------------------------------------------------------------------\n\x1b[33mtests/test_harness.exodiff: \x1b[0m| Moose Test Performance: Alive time=0.10216, Active time=0.07228                                                         |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m -------------------------------------------------------------------------------------------------------------------------\n\x1b[33mtests/test_harness.exodiff: \x1b[0m| Event                                      nCalls     Total Time  Avg Time    Total Time  Avg Time    % of Active Time  |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|                                                       w/o Sub     w/o Sub     With Sub    With Sub    w/o S    With S   |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|-------------------------------------------------------------------------------------------------------------------------|\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|                                                                                                                         |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|                                                                                                                         |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m| Application                                                                                                             |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   Full Runtime                             1          0.0022      0.002170    0.0723      0.072296    3.00     100.02   |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|                                                                                                                         |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m| Execution                                                                                                               |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   computeKernels()                         38         0.0023      0.000060    0.0023      0.000060    3.14     3.14     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   computeNodalBCs()                        38         0.0003      0.000009    0.0003      0.000009    0.45     0.45     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   compute_jacobian()                       5          0.0006      0.000129    0.0006      0.000129    0.89     0.89     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   compute_residual()                       38         0.0023      0.000060    0.0049      0.000128    3.14     6.75     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   solve()                                  4          0.0102      0.002542    0.0157      0.003924    14.07    21.71    |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|                                                                                                                         |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m| Output                                                                                                                  |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   CSV::output()                            5          0.0000      0.000005    0.0000      0.000005    0.04     0.04     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   Exodus::output()                         5          0.0111      0.002211    0.0111      0.002211    15.30    15.30    |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|                                                                                                                         |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m| Setup                                                                                                                   |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   Application Setup                        1          0.0390      0.039043    0.0421      0.042058    54.02    58.19    |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   FEProblemBase::init::meshChanged()       1          0.0001      0.000137    0.0001      0.000137    0.19     0.19     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   Initial updateActiveSemiLocalNodeRange() 1          0.0000      0.000015    0.0000      0.000015    0.02     0.02     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   Initial updateGeomSearch()               2          0.0000      0.000002    0.0000      0.000002    0.01     0.01     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   NonlinearSystem::update()                1          0.0002      0.000179    0.0002      0.000179    0.25     0.25     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   eq.init()                                1          0.0027      0.002698    0.0027      0.002698    3.73     3.73     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   execMultiApps()                          1          0.0000      0.000010    0.0000      0.000010    0.01     0.01     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   execTransfers()                          1          0.0000      0.000006    0.0000      0.000006    0.01     0.01     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   initial adaptivity                       1          0.0000      0.000000    0.0000      0.000000    0.00     0.00     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   initialSetup()                           1          0.0011      0.001077    0.0013      0.001290    1.49     1.78     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   reinit() after updateGeomSearch()        1          0.0000      0.000003    0.0000      0.000003    0.00     0.00     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|                                                                                                                         |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m| Utility                                                                                                                 |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m|   projectSolution()                        1          0.0002      0.000174    0.0002      0.000174    0.24     0.24     |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m -------------------------------------------------------------------------------------------------------------------------\n\x1b[33mtests/test_harness.exodiff: \x1b[0m| Totals:                                    147        0.0723                                          100.00            |\n\x1b[33mtests/test_harness.exodiff: \x1b[0m -------------------------------------------------------------------------------------------------------------------------\n\x1b[33mtests/test_harness.exodiff: \x1b[0mRunning exodiff: /opt/civet/build_0/moose/framework/contrib/exodiff/exodiff -m  -F 1e-10 -t 5.5e-06  /opt/civet/build_0/moose/test/tests/test_harness/gold/exodiff_out.e /opt/civet/build_0/moose/test/tests/test_harness/exodiff_out.e\n\x1b[33mtests/test_harness.exodiff: \x1b[0mERROR: \n\x1b[33mtests/test_harness.exodiff: \x1b[0m   *****************************************************************\n\x1b[33mtests/test_harness.exodiff: \x1b[0m              EXODIFF\t(Version: 2.90) Modified: 2018-02-15\n\x1b[33mtests/test_harness.exodiff: \x1b[0m              Authors:  Richard Drake, rrdrake@sandia.gov           \n\x1b[33mtests/test_harness.exodiff: \x1b[0m                        Greg Sjaardema, gdsjaar@sandia.gov          \n\x1b[33mtests/test_harness.exodiff: \x1b[0m              Run on    2018/03/28   18:14:08 MDT\n\x1b[33mtests/test_harness.exodiff: \x1b[0m   *****************************************************************\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\n\x1b[33mtests/test_harness.exodiff: \x1b[0mexodiff: ERROR: Couldn\'t open file "/opt/civet/build_0/moose/test/tests/test_harness/exodiff_out.e".\n\x1b[33mtests/test_harness.exodiff: \x1b[0mReading first file ... \n\x1b[33mtests/test_harness.exodiff: \x1b[0mReading second file ... \n\x1b[33mtests/test_harness.exodiff: \x1b[0mexodiff: ERROR: exodiff: ERROR: Couldn\'t open file "/opt/civet/build_0/moose/test/tests/test_harness/exodiff_out.e". File exists, but is not an exodusII file.\n\x1b[33mtests/test_harness.exodiff: \x1b[0m \n\x1b[33mtests/test_harness.exodiff: \x1b[0m################################################################################\n\x1b[33mtests/test_harness.exodiff: \x1b[0mTester failed, reason: EXODIFF\n\x1b[33mtests/test_harness.exodiff: \x1b[0m\ntests/test_harness.exodiff \x1b[90m..................................................................\x1b[0m \x1b[33mFAILED (EXODIFF)\x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mWorking Directory: /opt/civet/build_0/moose/test/tests/test_harness\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mRunning command: /opt/civet/build_0/moose/test/moose_test-opt -i csvdiff.i --error --error-unused --error-override --no-gdb-backtrace\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mFramework Information:\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mMOOSE Version:           git commit b22a48e on 2018-03-28\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mLibMesh Version:         727d0685dc96e841c5805608653410ea0cd926a6\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mPETSc Version:           3.7.6\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mCurrent Time:            Wed Mar 28 18:14:08 2018\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mExecutable Timestamp:    Wed Mar 28 17:52:22 2018\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mParallelism:\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Num Processors:          1\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Num Threads:             1\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mMesh: \n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Parallel Type:           replicated\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Mesh Dimension:          2\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Spatial Dimension:       2\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Nodes:                   \n\x1b[31mtests/test_harness.csvdiff: \x1b[0m    Total:                 9\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m    Local:                 9\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Elems:                   \n\x1b[31mtests/test_harness.csvdiff: \x1b[0m    Total:                 4\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m    Local:                 4\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Num Subdomains:          1\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Num Partitions:          1\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mNonlinear System:\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Num DOFs:                9\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Num Local DOFs:          9\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Variables:               "u" \n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Finite Element Types:    "LAGRANGE" \n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Approximation Orders:    "FIRST" \n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mExecution Information:\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Executioner:             Transient\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  TimeStepper:             ConstantDT\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Solver Mode:             Preconditioned JFNK\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m  Preconditioner:          hypre boomeramg \n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mTime Step  0, time = 0\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m                dt = 0\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mPostprocessor Values:\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m| time           | num_dofs       |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   0.000000e+00 |   0.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mTime Step  1, time = 0.1\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m                dt = 0.1\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m 0 Nonlinear |R| = \x1b[32m1.732051e+00\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      0 Linear |R| = \x1b[32m1.732051e+00\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      1 Linear |R| = \x1b[32m6.094240e-02\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      2 Linear |R| = \x1b[32m3.918388e-04\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      3 Linear |R| = \x1b[32m6.663069e-16\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m 1 Nonlinear |R| = \x1b[32m4.747245e-09\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\x1b[32m Solve Converged!\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mPostprocessor Values:\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m| time           | num_dofs       |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   0.000000e+00 |   0.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   1.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mTime Step  2, time = 0.2\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m                dt = 0.1\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m 0 Nonlinear |R| = \x1b[32m1.640283e-01\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      0 Linear |R| = \x1b[32m1.640283e-01\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      1 Linear |R| = \x1b[32m2.793150e-03\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      2 Linear |R| = \x1b[32m4.100195e-05\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      3 Linear |R| = \x1b[32m3.334487e-13\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m 1 Nonlinear |R| = \x1b[32m6.897799e-11\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\x1b[32m Solve Converged!\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mPostprocessor Values:\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m| time           | num_dofs       |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   0.000000e+00 |   0.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   1.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   2.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mTime Step  3, time = 0.3\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m                dt = 0.1\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m 0 Nonlinear |R| = \x1b[32m1.464539e-01\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      0 Linear |R| = \x1b[32m1.464539e-01\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      1 Linear |R| = \x1b[32m2.493884e-03\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      2 Linear |R| = \x1b[32m3.660888e-05\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      3 Linear |R| = \x1b[32m1.886645e-12\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m 1 Nonlinear |R| = \x1b[32m1.250490e-10\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\x1b[32m Solve Converged!\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mPostprocessor Values:\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m| time           | num_dofs       |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   0.000000e+00 |   0.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   1.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   2.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   3.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mTime Step  4, time = 0.4\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m                dt = 0.1\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m 0 Nonlinear |R| = \x1b[32m1.307624e-01\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      0 Linear |R| = \x1b[32m1.307624e-01\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      1 Linear |R| = \x1b[32m2.226682e-03\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      2 Linear |R| = \x1b[32m3.268650e-05\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      3 Linear |R| = \x1b[32m7.041399e-13\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m 1 Nonlinear |R| = \x1b[32m8.437642e-11\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\x1b[32m Solve Converged!\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mPostprocessor Values:\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m| time           | num_dofs       |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   0.000000e+00 |   0.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   1.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   2.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   3.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   4.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mTime Step  5, time = 0.5\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m                dt = 0.1\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m 0 Nonlinear |R| = \x1b[32m1.167521e-01\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      0 Linear |R| = \x1b[32m1.167521e-01\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      1 Linear |R| = \x1b[32m1.988109e-03\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      2 Linear |R| = \x1b[32m2.918438e-05\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m      3 Linear |R| = \x1b[32m1.385863e-12\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m 1 Nonlinear |R| = \x1b[32m4.716458e-11\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\x1b[32m Solve Converged!\x1b[39m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mPostprocessor Values:\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m| time           | num_dofs       |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   0.000000e+00 |   0.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   1.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   2.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   3.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   4.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   5.000000e-01 |   9.000000e+00 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m+----------------+----------------+\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m -------------------------------------------------------------------------------------------------------------------------\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m| Moose Test Performance: Alive time=0.088975, Active time=0.065209                                                       |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m -------------------------------------------------------------------------------------------------------------------------\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m| Event                                      nCalls     Total Time  Avg Time    Total Time  Avg Time    % of Active Time  |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|                                                       w/o Sub     w/o Sub     With Sub    With Sub    w/o S    With S   |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|-------------------------------------------------------------------------------------------------------------------------|\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|                                                                                                                         |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|                                                                                                                         |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m| Application                                                                                                             |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   Full Runtime                             1          0.0027      0.002662    0.0652      0.065220    4.08     100.02   |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|                                                                                                                         |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m| Execution                                                                                                               |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   computeKernels()                         40         0.0027      0.000067    0.0027      0.000067    4.12     4.12     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   computeNodalBCs()                        40         0.0004      0.000010    0.0004      0.000010    0.60     0.60     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   computeUserObjects(TIMESTEP_END)         5          0.0000      0.000008    0.0000      0.000008    0.06     0.06     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   compute_jacobian()                       5          0.0007      0.000143    0.0007      0.000143    1.10     1.10     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   compute_residual()                       40         0.0027      0.000067    0.0058      0.000144    4.12     8.83     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   solve()                                  5          0.0126      0.002518    0.0191      0.003815    19.31    29.25    |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|                                                                                                                         |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m| Output                                                                                                                  |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   CSV::output()                            6          0.0004      0.000062    0.0004      0.000062    0.57     0.57     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|                                                                                                                         |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m| Setup                                                                                                                   |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   Application Setup                        1          0.0387      0.038665    0.0416      0.041649    59.29    63.87    |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   FEProblemBase::init::meshChanged()       1          0.0001      0.000142    0.0001      0.000142    0.22     0.22     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   Initial updateActiveSemiLocalNodeRange() 1          0.0000      0.000015    0.0000      0.000015    0.02     0.02     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   Initial updateGeomSearch()               2          0.0000      0.000002    0.0000      0.000002    0.01     0.01     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   NonlinearSystem::update()                1          0.0002      0.000191    0.0002      0.000191    0.29     0.29     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   eq.init()                                1          0.0027      0.002650    0.0027      0.002650    4.06     4.06     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   execMultiApps()                          1          0.0000      0.000011    0.0000      0.000011    0.02     0.02     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   execTransfers()                          1          0.0000      0.000006    0.0000      0.000006    0.01     0.01     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   initial adaptivity                       1          0.0000      0.000001    0.0000      0.000001    0.00     0.00     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   initialSetup()                           1          0.0012      0.001178    0.0014      0.001418    1.81     2.17     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   reinit() after updateGeomSearch()        1          0.0000      0.000004    0.0000      0.000004    0.01     0.01     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|                                                                                                                         |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m| Utility                                                                                                                 |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m|   projectSolution()                        1          0.0002      0.000198    0.0002      0.000198    0.30     0.30     |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m -------------------------------------------------------------------------------------------------------------------------\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m| Totals:                                    155        0.0652                                          100.00            |\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m -------------------------------------------------------------------------------------------------------------------------\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mRunning CSVDiffer.py\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mIn /opt/civet/build_0/moose/test/tests/test_harness/csvdiff_out.csv: File does not exist!\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m################################################################################\n\x1b[31mtests/test_harness.csvdiff: \x1b[0mTester failed, reason: FILE DOES NOT EXIST\n\x1b[31mtests/test_harness.csvdiff: \x1b[0m\ntests/test_harness.csvdiff \x1b[90m......................................................\x1b[0m \x1b[31mFAILED (FILE DOES NOT EXIST)\x1b[0m\n\n\nFinal Test Results:\n--------------------------------------------------------------------------------------------------------------\ntests/test_harness.csvdiff \x1b[90m......................................................\x1b[0m \x1b[31mFAILED (FILE DOES NOT EXIST)\x1b[0m\ntests/test_harness.exodiff \x1b[90m..................................................................\x1b[0m \x1b[33mFAILED (EXODIFF)\x1b[0m\n--------------------------------------------------------------------------------------------------------------\nRan 2 tests in 1.6 seconds\n\x1b[1m0 passed\x1b[0m, \x1b[1m0 skipped\x1b[0m, \x1b[1m0 pending\x1b[0m, \x1b[1m\x1b[31m2 FAILED\x1b[0m\n'
TestHarness/tests.diff: 
TestHarness/tests.diff: ----------------------------------------------------------------------
TestHarness/tests.diff: Ran 1 test in 2.362s
TestHarness/tests.diff: 
TestHarness/tests.diff: FAILED (failures=1)
TestHarness/tests.diff: 
TestHarness/tests.diff: ################################################################################
TestHarness/tests.diff: Tester failed, reason: CRASH
TestHarness/tests.diff: 
    FAIL TestHarness/tests.diff FAILED (CRASH)
```

--- a/python/TestHarness/tests/test_WriteResults.py
+++ b/python/TestHarness/tests/test_WriteResults.py
@@ -23,6 +23,12 @@ class TestHarnessTester(TestHarnessTestCase):
         except:
             pass
 
+    def tearDown(self):
+        """
+        tearDown occurs after every test.
+        """
+        self.setUp()
+
     def testWriteOK(self):
         """ Test ability to write separate OK test --sep-files-ok """
         self.runTests('--no-color', '-i', 'always_ok', '--sep-files-ok', '--output-dir', self.output_dir)

--- a/python/TestHarness/tests/tests
+++ b/python/TestHarness/tests/tests
@@ -106,5 +106,6 @@
   [./write_results]
     type = PythonUnitTest
     input = test_WriteResults.py
+    prereq = 'diff'
   [../]
 []

